website/content/docs/v0.15/Guides/resetting-a-machine.md

```diff
@@ -25,3 +27,11 @@ The `graceful` flag is especially important when considering HA vs. non-HA Talos
 If the machine is part of an HA cluster, a normal, graceful reset should work just fine right out of the box as long as the cluster is in a good state.
 However, if this is a single node cluster being used for testing purposes, a graceful reset is not an option since Etcd cannot be "left" if there is only a single member.
 In this case, reset should be used with `--graceful=false` to skip performing checks that would normally block the reset.
+
+## Kernel Parameter
+
+Another way to reset a machine is to specify `talos.experimental.wipe=system` kernel parameter.
+If the machine got stuck in the boot loop and you access to the console you can use GRUB to specify this kernel argument.
+Then when Talos boots for the next time it will reset system disk and reboot.
+
+Next steps can be to install Talos either usign PXE boot or by mounting an ISO.
```

Comment:
could you please also add it to https://github.com/talos-systems/talos/blob/05fd16adfde59550b3f8982262319250871390f7/website/content/docs/v0.15/Reference/kernel.md

Reply:
There is a typo in the last sentence: usign instead of using 😉